.github/licenserc.yaml

```diff
@@ -0,0 +1,45 @@
+header:
+  license:
+    spdx-id: Apache-2.0
+    content: |
+      SPDX-FileCopyrightText: 2024-2026 SAP SE or an SAP affiliate company and Greenhouse contributors
+      SPDX-License-Identifier: Apache-2.0
+
+  paths: # `paths` are the path list that will be checked (and fixed) by license-eye, default is ['**'].
+    - '**'
+
+  paths-ignore:
+    - '.github/**'
+    - 'charts/**'
+    - '.reuse /dep5'
+    - 'LICENSES/*.txt'
+    - 'grafana/*.json'
+    - 'website/scripts/**'
+    - 'hack/**'
+    - 'docs/**'
+    - '**/*.md'
+    - 'LICENSE'
+    - 'NOTICE'
+    - 'PROJECT'
+    - '**/*.gitignore'
+    - '**/go.mod'
+    - '**/go.sum'
+    - '**/*.lock'
+    - '**/*.json'
+    - '**/.gitkeep'
+    - '**/*.jsx'
```

Comment:
@kengou I just saw that you excluded .jsx files from being commented. What is the reason for that? We have a lot of files in our project which are written in .jsx files.

Reply:
Good catch @hodanoori! `.jsx` files should not be excluded

Reply:
Fixing it in #27